test/Microsoft.DotNet.ProjectModel.Tests/LockFileReaderTests.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Linq;
+using FluentAssertions;
+using Microsoft.DotNet.ProjectModel.Graph;
+using Microsoft.DotNet.Tools.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.DotNet.ProjectModel.Tests
+{
+    public class LockFileReaderTests : TestBase
+    {
+        [Fact]
+        public void ReadLockFileReadsLibrariesWithPathProperty()
```

Comment:
The name here is weird, you put an emphasis on Path property, however you end up checking for everything in the asserts.